src/styles/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  width: 166px;
```

Comment:
add `box-sizing: border-box` so you wouldn't have to calculate this width and make it 200px